cmd/agent-protocol-forwarder/main.go

```diff
@@ -67,6 +69,12 @@ func (cfg *Config) Setup() (cmd.Starter, error) {
 		cfg.tlsConfig = &tlsConfig
 	}
 
+	cmd.ShowVersion(programName)
+
+	if showVersion {
+		cmd.Exit(0)
```

Comment:
nit: a question for my understanding. I see that Exit is mapped to os.Exit. Is there any benefit of doing this ?

Reply:
It is for unit tests. We can replace `Exit` in unit tests, and can check whether an exit code is expected one or not.

https://github.com/confidential-containers/cloud-api-adaptor/blob/f3f604e9cd8be01a1922b8c84e95bd4afd4b0bad/cmd/parse_test.go#L19-L23

Go does not provide a simple solution for testing `os.Exit` as described here.
https://stackoverflow.com/q/26225513

One possible solution was to use [Go monkeypatching](https://github.com/bouk/monkey), but it does not support s390x, and now it is archived.

It originally comes from my internal project. I don't think we need it unless we really need full code coverage in unit tests.